README.md

```diff
@@ -1,3 +1,43 @@
 # Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+#README
```

Comment:
When establishing headers, we want to put a space between the # and the word in order to create the header. Without a space, it does not create the header and inserts the "#" into the file

Reply:
Space added